src/demoserver.cpp

```diff
@@ -233,9 +233,39 @@ void DemoServer::handle_packet(AOPacket packet)
             QString packet = "CT#DEMO#" + tr("min_wait is deprecated. Use the client Settings for minimum wait instead!") + "#1#%";
             client_sock->write(packet.toUtf8());
         }
+        else if (contents[1].startsWith("/debug"))
+        {
+            QStringList args = contents[1].split(" ");
+            if (args.size() > 1)
+            {
+              bool ok;
+              int toggle = args.at(1).toInt(&ok);
+              if (ok && (toggle == 0 || toggle == 1)) {
```

Comment:
`ok` variable is redundant as `toggle` would be -1 on error and that is already checked by your if statement

Reply:
wouldn't .at(1) be an error if args size is less than 1?

Reply:
`args.size() > 1` holds true in this branch of code so it will not fail

also i'm clearly losing brain cells because toInt() returns 0 on failure, not -1